test/debugging_tests.jl

```diff
@@ -0,0 +1,172 @@
+# REopt®, Copyright (c) Alliance for Sustainable Energy, LLC. See also https://github.com/NREL/REopt.jl/blob/master/LICENSE.
```

Comment:
could be worth adding this to the changelog and explaining how it should be used?

Reply:
I wasn't planning on leaving this. I was going to migrate this test over to runtests.jl.